src/lib.rs

```diff
@@ -20,6 +21,7 @@ pub mod serde;
 pub mod sql;
 #[cfg(feature = "test-util")]
 pub mod test;
+pub mod validation_mode;
```

Comment:
`ValidationMode` can be easily moved to another module. Let's re-export it on the top level instead (or even better via `::query` if we want to have dedicated types for selects and inserts).